staker/commands.go

```diff
@@ -26,6 +26,14 @@ type stakingRequestCmd struct {
 	pop                     *cl.BabylonPop
 	errChan                 chan error
 	successChan             chan *chainhash.Hash
+	// Expansion-specific fields for Babylon integration
+	stakeExpansion *stakeExpansionReqFields
+}
+
+type stakeExpansionReqFields struct {
```

Comment:
we can create new constructor for `newOwnedStakeExpansionCommand` which takes all necessary fields